cypress/fixtures/questionnaire_import/questionnaire-template-sample.yaml

```diff
@@ -0,0 +1,50 @@
+name: Questionnaire Template   ( Sample )
```

Comment:
**question:** This name has 3 whitespaces between "Template" and "( Sample )", but the constant you created only has one. 

Is that expected?

Reply:
yes this is intentional, to check if it is handled corerctly when imported